src/lib/builder.ts

```diff
@@ -184,5 +233,21 @@ export async function build() {
     JSON.stringify(packageJson, null, 2)
   );
 
+  // https://vercel.com/docs/build-output-api/v3
+  // So far, only static sites are supported.
+  createVercelOutput(
```

Comment:
We'll make it configurable with options once we have more sophisticated CLI, but for now it's enabled by default.